signalfx/resource_signalfx_org_token.go

```diff
@@ -347,6 +353,49 @@ func orgTokenAPIToTF(d *schema.ResourceData, t *orgtoken.Token) error {
 		return err
 	}
 
+	secret := func() string {
+		if d.Get("store_secret").(bool) {
+			return t.Secret
+		}
+		return ""
+	}()
```

Comment:
I would prefer not needing to lambda function here in order to get the desired behaviour since it makes the stack frames header to read if sent to support.